examples/tensorflow/README.md

```diff
@@ -4,8 +4,6 @@ This resource gives you a starting example for running TensorFlow training tasks
 
 [single-gpu-tensorflow](single-gpu-tensorflow.ipynb) train your model with TensorFlow on a single GPU. 
 
-[multi-gpu-tensorflow](multi-gpu-tensorflow.ipynb) train your model with TensorFlow on multiple GPUs. 
-
 
 
 ## Extra Resources
```

Comment:
You should have the extra resources link to the doc page on multigpu saturn cloud, once that exists.

Reply:
Will add this , once I merge this pr and create docs for these examples .